### PR TITLE
MR-627 Cypress filter by ID

### PR DESCRIPTION
**Pull request by wavebeem**
- [x] This change is documented in CHANGELOG.md
- [x] This change has been tested via the terms in TESTING.md

### CODE REVIEW

File: cypress/integration/filter-by-id.spec.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+describe("Filter by ID", () => {
+  it("should only show data specified by tags.filter and placemarks.filter", () => {
+    cy.visit("/cypress/filter-by-id");
+    cy.get('[data-meridian-tag-id="546C0E032A87"]').should("exist");
+    cy.get("[data-meridian-tag-id]").should("have.length", 1);
+    cy.get('[data-meridian-placemark-id="5717271485874176"]').should("exist");
```
Comment (wavebeem) [inline cypress/integration/filter-by-id.spec.ts:6]: I noticed this is just the "second half" of the placemark ID (`5755685136498688_5717271485874176`) which is kinda weird... but it's shorter at least. Our IDs are a total mess.

Comment (vercasson) [reply cypress/integration/filter-by-id.spec.ts:6]: The second part is the placemark ID as it exists in the DB

File: demo/cypress/filter-by-id/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        locationID: "5198682008846336",
+        floorID: "5755685136498688",
+        height: "100%",
+        tags: {
+          filter: tag => tag.id === "546C0E032A87"
+        },
+        placemarks: {
+          filter: placemark =>
+            placemark.id === "5755685136498688_5717271485874176"
+        },
```
Comment (vercasson) [inline demo/cypress/filter-by-id/index.html:44]: Sorry about leaving this in the basic test. The click handlers below do nothing and aren't needed.

File: cypress/integration/filter-by-id.spec.ts (hunk 1 above)
Comment (wavebeem) [reply cypress/integration/filter-by-id.spec.ts:6]: I know, I'm just saying I regret that customers can see that "bare" ID anywhere... it's confusing to have two forms of the same concept being tossed around with the same name 😢

**Review by vercasson — APPROVED** [review]

**Pull request merged**